core/src/main/resources/lib/form/radioBlock/radioBlock.js

```diff
@@ -18,7 +18,13 @@ var radioBlockSupport = {
             n = blockStart;
         }
         while((n = n.next()) != blockEnd) {
-          n.style.display = show ? "" : "none";
+            if (show) {
+                n.style.visibility = "visible";
+                n.style.position = "";
+            } else {
+                n.style.visibility = "hidden";
+                n.style.position = "absolute";
```

Comment:
```suggestion
```

Reply:
Hello @romenrg , thanks for the suggestion! The reason why I set position to 'absolute' is that if there's something in the foldable block, there will reserve an empty space. See demo
![o](https://user-images.githubusercontent.com/20413055/93464815-5ba28700-f91c-11ea-9cca-4cd80b661008.gif)
So I change the position to 'absolute' to clear the empty space.

Reply:
I see, @benebsiny . Thank you for the explanation!
Then ignore my suggestion of removing the position attribute change. Just maybe consider the "static" one instead of the "", although that's not a big deal.

Reply:
I've update. Have a look at it.

Reply:
Awesome. Thanks!